openshift/wps-api-base/docker/Dockerfile

```diff
@@ -34,12 +23,19 @@ ENV DEBIAN_FRONTEND=noninteractive
 # - gdal (for geospatial)
 # - R (for cffdrs)
 # - xfonts-75dpi, xfonts-base (for wkhtmltopdf)
-RUN apt-get update --fix-missing && apt-get -y install python3 python3-pip python3-dev python-is-python3 libgdal-dev r-base xfonts-base xfonts-75dpi
+RUN apt-get update --fix-missing && apt-get -y install python3 python3-pip python3-dev python-is-python3 libgdal-dev r-base xfonts-base xfonts-75dpi curl
+
+RUN curl -sSL https://install.python-poetry.org > /tmp/install.python-poetry.org
+RUN curl -sSL https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6.1-2/wkhtmltox_0.12.6.1-2.jammy_amd64.deb > /tmp/wkhtmltox_0.12.6.1-2.jammy_amd64.deb
+# Enable for mac M1
+# RUN curl -sSL https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6.1-2/wkhtmltox_0.12.6.1-2.jammy_arm64.deb > /tmp/wkhtmltox_0.12.6.1-2.jammy_amd64.deb
+
 
 # - wkhtmltopdf (for making pdf's)
-COPY --from=download /tmp/wkhtmltox_0.12.6.1-2.jammy_amd64.deb /tmp/wkhtmltox_0.12.6.1-2.jammy_amd64.deb
-RUN dpkg -i /tmp/wkhtmltox_0.12.6.1-2.jammy_amd64.deb
 
+# Enable for mac M1
+# RUN dpkg -i /tmp/wkhtmltox_0.12.6.1-2.jammy_amd64.deb
```

Comment:
Is lien 37 needed? It looks the same as 28. Or perhaps it was supposed to be 'arm64.deb'?

Reply:
Yep, should be arm64, thanks!

Reply:
Fixed